resources/environment.go

```diff
@@ -74,6 +74,7 @@ func environmentServiceDefinition(environment *Environment, application *Applica
 				"envUUID":      environment.UUID,
 			},
 		},
+		//@TODO allow for multiple ports creations
```

Comment:
I think having a default port now is fine because a port entry is not required for creating a version.